tests/Unit/API/Trace/SpanContextValidatorTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\API\Unit\Trace;
+
+use OpenTelemetry\API\Trace\SpanContext;
+use OpenTelemetry\API\Trace\SpanContextValidator;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\API\Trace\SpanContextValidator
+ */
+class SpanContextValidatorTest extends TestCase
+{
+    private const TRACE_ID = 'ff000000000000000000000000000041';
+    private const SPAN_ID = 'ff00000000000041';
+
+    public function test_is_invalid_trace_id(): void
```

Comment:
Remove is from the test name for all tests.

Reply:
done